vixenconverter/vixenfiles/VixenProfile.py

```diff
@@ -37,7 +37,7 @@ def get_channels(self):
             'channelName': channel.text.replace('(', ' ').replace(')', ' ').replace('.', ' ').replace('\'', ' '),
             'color': format(int(channel.attrib['color']) & 0xffffff, "06X"),
             'internalChannel': output_order.index(int(channel.attrib['output'])) + 1,
-            'dmxChannel': 121, # Unused DMX channel in the 2016 show
+            'dmxChannel': 511, # Unused DMX channel
```

Comment:
Thought: instead of changing this in hardcode every year, why not pass in a dummy channel to the library and have the outer program get it from up the stack, like a config file or CLI parameter?

Reply:
I liked this idea, so I did it. Still look good @computergeek125 ?